lib/tasks/reports.rake

```diff
@@ -120,3 +120,96 @@ namespace :reports do
     process_notifications :monthly
   end
 end
+
+# TRANSLATORS: do not translate
+desc <<~END_DESC
+  Generate dummy reports and display performance numbers. Half of lines are unique,
+  half are same. Reports are left in the database, they will be expired via cron or
+  can be expired manually or by deleting all hosts named hostXX.example.com.
+
+  Available conditions:
+    * origin           => Puppet or Ansible
+    * hosts            => number of newly created hosts
+    * reports          => number of reports per host
+    * lines            => number of lines per report
+
+    Example:
+      rake reports:generate hosts=20 reports=100 lines=200
+END_DESC
+namespace :reports do
+  task :generate => :environment do
```

Comment:
We have [forgeries](https://github.com/theforeman/foreman/blob/develop/lib/tasks/seed.rake). Perhaps it would be better to implement it there?

Reply:
Good point, I will move it there.